local_server/common/config/dataset_config.py

```diff
@@ -4,7 +4,7 @@
 from local_server.common.annotations.local_file_csv import AnnotationsLocalFile
 from local_server.common.config.base_config import BaseConfig
 from local_server.common.errors import ConfigurationError, OntologyLoadFailure, AnnotationsError
-from local_server.compute.scanpy import get_scanpy_module
+from shared_utils.compute.scanpy import get_scanpy_module
```

Comment:
I am fairly certain the scanpy-dependent features (re-embedding) are only supported in the local/pip-installed back-end, and are not destined for the hosted (or at least not with the same scanpy-derived implementation).  So `scanpy.py` is probably not shared/common code.